RuriLib/Models/Blocks/Custom/ScriptBlockInstance.cs

```diff
@@ -174,24 +175,28 @@ public override string ToCSharp(List<string> definedVariables, ConfigSettings se
                     break;
 
                 case Interpreter.NodeJS:
+                    // for @Kermit with <3
                     var nodeScript = @$"module.exports = async ({MakeInputs()}) => {{
-{Script}
-var noderesult = {{
-{MakeNodeObject()}
-}};
-return noderesult;
-}}";
-
-                    scriptHash = HexConverter.ToHexString(Crypto.MD5(Encoding.UTF8.GetBytes(nodeScript)));
+                        {Script}
+                        var noderesult = {{
+                        {MakeNodeObject()}
+                        }};
+                        return noderesult;
+                        }}";
+                    string a = @"DID YOU REALLY TRY TO 
```

Comment:
This is unprofessional, please remove. Also, writing something to a file is not needed anymore. We will keep the duality in the `InvokeNode<T>` method for backwards compatibility, in case of configs that were created with the old versions of RuriLib, but it's not required to write anything to a script file anymore, so it can all be removed. Only the `escapedScript` should be kept and directly injected into the code.